Readability.js

```diff
@@ -680,6 +688,7 @@ Readability.prototype = {
         if (stripUnlikelyCandidates) {
           if (this.REGEXPS.unlikelyCandidates.test(matchString) &&
               !this.REGEXPS.okMaybeItsACandidate.test(matchString) &&
+              !this._isMaybeAncestorACandidate(node, 3) &&
```

Comment:
That said, I don't know if 3 is the right depth here... but it can be a starting point, and we can always expand it if we think it's necessary.